app/webroot/js/misp2.4.50.js

```diff
@@ -2489,3 +2494,23 @@ $(".queryPopover").click(function() {
 	});
 });
 
+function requestAPIAccess() {
+	var destination = 'users';
+	var action = 'request_API';
+	url = "/" + destination + "/" + action + "/";
```

Comment:
Which is fine, if it was a parametrised generic function I'd be all for it. For single use functions such as this it doesn't make sense.